src/app/garfile/customs/index.njk

```diff
@@ -67,57 +68,52 @@
       </div>
 
       <div class="govuk-form-group {{ m.form_group_class('freeCirculation', errors) }}" id="freeCirculation">
-        <fieldset class="govuk-fieldset">
+        <fieldset class="govuk-fieldset" aria-describedby="freeCirculation-hint">
           <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
             <h2 class="govuk-fieldset__heading">{{__('field_free_circulation')}}</h2>
-            <span id="free-circulation-hint" class="govuk-hint">{{__('field_free_circulation_hint')}}</span>
+            <span id="freeCirculation-hint" class="govuk-hint">{{__('field_free_circulation_hint')}}</span>
+            {{ m.error_message('freeCirculation', errors, __) }}
           </legend>
           <div class="govuk-radios">
           {% for choice in freeCirculationOptions %}
           <div class="govuk-radios__item">
-              <input class="govuk-radios__input" id="{{choice.option}}" type="radio" name="freeCirculation" value="{{choice.value}}"
+            <input id="{{choice.option}}" name="freeCirculation" class="govuk-radios__input" type="radio"
+              value="{{choice.value}}" aria-labelledby="freeCirculation-label-{{ loop.index }}"
```

Comment:
Hopefully the aria-labelledby will allow a screen reader to read out the label for the radio buttons.